### ISSUE TITLE
Using Keyframe Mixin (Compass) produces a Fatal

### ISSUE THREAD

**Issue by wernerdweight** [tonyganch/gonzales-pe#205]
This code (which is valid sass (using compass)):

``` sass
+keyframes(loadMoreSpinner)
    0%
        +transform(rotate(0deg) scale(1))
    50%
        +transform(rotate(180deg) scale(1.1))
    100%
        +transform(rotate(360deg) scale(1))
```

produces this error:

``` bash
2:1  error  Please check validity of the block starting from line #2  Fatal
```

I'm using it via Sass-Lint (originally posted this issue here: https://github.com/sasstools/sass-lint/issues/797)

**Comment by raphaelfabeni** [comment]
I have a similar error 😢 . I posted about it in the same issue that @wernerdweight .
Link here => [sasstols/sass-lint#797](https://github.com/sasstools/sass-lint/issues/797#issuecomment-257749983)

**Comment by raphaelfabeni** [comment]
I have a similar error using `.sass` syntax. 😢 
I posted in the same issue. sasstols/sasslint#797

**Comment by raphaelfabeni** [comment]
I have a similar error using .sass syntax. 😢 
I posted in the same issue. sasstools/sass-lint#797.

**Comment by tonyganch** [comment]
@raphaelfabeni, I took a look at example you posted at sass-lint, and the issue is somewhat not related to @wernerdweight's one. The thing that breaks your code should be fixed with PR #214, so hold on a little.
